fullstack-cert/css-projects/pricing-plans-layout/styles.css

```diff
@@ -0,0 +1,129 @@
+body {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-direction: column;
+  min-height: 100vh;
+}
+
+h1 {
+  margin-bottom: 20px;
+}
+
+h2,
+p {
+  margin: 0;
+}
+
+.pricing-container {
+  display: flex;
+  flex-wrap: wrap;
+  align-content: center; /* controls vertical spacing if there are multiple rows */
+  border: 2px solid #000;
+  gap: 20px;
+  width: 80%;
+  max-width: 900px;
+  height: 70vh; /* Allows align-content effect to show */
+  padding: 0 20px;
+}
+
+.pricing-card {
+  background: rgb(247, 247, 247);
+  box-shadow: 0 4px 8px rgba(0, 0, 0, 0.1);
+  padding: 20px;
+  flex: 0 0 200px; /* sets the initial flex-grow to 0, initial flex-shrink to 0, and flex-basis to 200px */
```

Comment:
this seem a comment for a workshop, in a lab we can't do this step-by-step approack